packages/crypto-react-native/src/utils.ts

```diff
@@ -29,9 +29,9 @@ export const binaryToByte = (bin: string) => parseInt(bin, 2)
 export const deriveChecksumBits = (entropyBuffer: Buffer | string) => {
   const ENT = entropyBuffer.length * 8
   const CS = ENT / 32
-  const hash = sha256(entropyBuffer)
+  const hash = sha256.digest(entropyBuffer)
```

Comment:
This was the fix for the checksums, `sha256(entropyBuffer)` returned a hex string representation but a byte array was expected.

Reply:
@tyler-whitman, seems good, thank you for investigating this. I had a feeling that there were still some things during the original @helium/crypto-react merge https://github.com/helium/helium-js/pull/234#issuecomment-955271811, but wasn't able to verify as I am without a computer. also, kudos on bringing the tests to the react package. have a good week!